cmd/proxy/actions/readiness.go

```diff
@@ -0,0 +1,24 @@
+package actions
+
+import (
+	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/storage"
+	"net"
+)
+
+func getReadinessHandler(s storage.Backend) buffalo.Handler {
+	return func(c buffalo.Context) error {
+		_, err := s.List(c, "github.com/gomods/athens")
+
+		if err != nil {
+			return c.Render(500, nil)
+		}
+
+		_, err = net.LookupIP("github.com")
```

Comment:
List goes to local storage as well as upstream so i guess in case this test is failing the previous one fails as well
Look here: https://github.com/gomods/athens/blob/b92a642f1ac49758d21781f3e27de70b2fc140c2/pkg/download/protocol.go#L62

Reply:
@michalpristas so, you suggesting removing DNS resolve test?

Reply:
yeah as long as list stays there